runtime/hydradx/src/evm/mod.rs

```diff
@@ -59,6 +63,8 @@ mod runner;
 pub use erc20_currency::Erc20Currency;
 pub use erc20_currency::Function;
 pub use executor::Executor;
+use hydra_dx_math::ema::EmaPrice;
+use hydradx_traits::NativePriceOracle;
 pub use primitives::AccountId as AccountIdType;
 
 // Current approximation of the gas per second consumption considering
```

Comment:
This approximation is outdated. We have 2sec execution. Do we want to keep it?

Reply:
not sure how it as appriximated back then, maybe @cl0w5 ?